### PR TITLE
Add the possibility of detecting the FOS 

### PR DESCRIPTION
**Pull request by fernandoGietz**
Detect the FOS OS when the client is doing a task
Add the cogs icon in green colour

### CODE REVIEW

File: packages/web/lib/fog/host.class.php [hunk 1]
```diff
@@ -2169,7 +2169,18 @@ public function getPingCodeStr()
...
                 case 111:
-                        printf($strtoupdate, 'linux', 'linux', 'blue', 'Linux');
+                        $taskID = self::getSubObjectIDs(
+                                'Task',
+                                array('hostID' => $this->get('id'),
+                                      'stateID' => 2
+                                ),
+                                'id'
+                        );
+                        if (is_null($taskID))
```
Comment (fernandoGietz) [inline packages/web/lib/fog/host.class.php]: When you open a socket to FOS client, the fsopensocket returns ths same code than linux. To make a diference is necesary add a new condition, the client has an active task (taskStatusID = 2, "In progress")

**Comment by Sebastian-Roth** [comment]
@fernandoGietz Is this ready to be merged? I don't have the time to fully test it but I assume this is working from looking at the code.

@mastacontrola What do you think?

**Comment by fernandoGietz** [comment]
I tested it in dev environment and works fine, I install the changes in my production environment and no problems.

**Comment by Sebastian-Roth** [comment]
@fernandoGietz Sorry, forgot to ask you in chat, should we merge this pull request or did you move all the changes to the new `dev-fog-plugins` branch and we should just close this PR?

**Comment by fernandoGietz** [comment]
Hi Sebastian,
You can merge it. I would like to merge only the changes in the host.class.php and the other discard them and begin to use the new branch "dev-fog-plugins" for the new plugin, but it is not possible, merge all.

**Comment by Sebastian-Roth** [comment]
@fernandoGietz Well it is possible. If you remove the other changes (files) from **your** current `dev-branch` and push out to https://github.com/fernandoGietz/fogproject/tree/dev-branch again we'll only see the changes in host.class.php left for this pull request.

**Comment by fernandoGietz** [comment]
Done!

**Comment by Sebastian-Roth** [comment]
@fernandoGietz Ok, nice. Merging now.

**Pull request merged**